pkg/commands/compute/pack_test.go

```diff
@@ -36,41 +34,17 @@ func TestPack(t *testing.T) {
 				"Creating .tar.gz file...",
 			},
 			expectedFiles: [][]string{
-				{"pkg", "mypackagename", "bin", "main.wasm"},
```

Comment:
ℹ️ Now we're cleaning up after ourselves we need to fix the tests to not check for that directory/files any more.